ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -438,13 +433,55 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
 
       Object.assign(groupToUpdate, {
         ...fractionalData,
-        ...(hasChanges && { action: ActionTypes.CHANGED })
+        ...(hasChanges && { action: isMhrTransfer ? ActionTypes.CHANGED : ActionTypes.CORRECTED })
       })
 
       const updatedOwnerGroups = [...homeOwnerGroups]
       setTransferOrRegistrationHomeOwnerGroups(updatedOwnerGroups)
     }
 
+  /**
+   * Return the baseline owners snapshot by id.
+   * @param ownerId The owner identifier
+   */
+  const getMhrBaselineOwnerById = (ownerId: number): MhrRegistrationHomeOwnerIF => {
+    let currentOwner: MhrRegistrationHomeOwnerIF
+    getMhrBaseline.value?.ownerGroups.forEach(group => group.owners.find(owner => {
+      if (owner.ownerId === ownerId) currentOwner = owner
+    }))
+
+    return currentOwner
+  }
+
+  /**
+   * Return the baseline owner group snapshot by id.
+   * @param groupId The group identifier
+   */
+  const getMhrBaselineOwnerGroupById = (groupId: number): MhrRegistrationHomeOwnerGroupIF => {
+    return getMhrBaseline.value?.ownerGroups[groupId-1]
+  }
+
+  /** Return true if the specified owner has been modified from current state **/
+  const isCorrectedOwner = (owner: MhrRegistrationHomeOwnerIF): boolean => {
+    const currentOwner = getMhrBaselineOwnerById(owner.ownerId)
+    const isEqualName = !!owner.individualName
+      ? deepChangesComparison(currentOwner.individualName, owner.individualName)
+      : deepChangesComparison(currentOwner.organizationName, owner.organizationName)
+
+    return currentOwner && (
+      isEqualName ||
+      deepChangesComparison(currentOwner.address, owner.address) ||
```

Comment:
It is not efficient to do a deep comparison for all props with the || operator. If, for example, the first prop, `currentOwner.address`, has changes, it would still have to go through all of the others. 
I think using .some() would be better, as it would stop iterating if any returns true. Eg. 
```
[ 'address', 'phoneNumber', ... ].some(prop => !deepChangesComparison(currentOwner[prop], owner[prop]))
```

Reply:
That is a pretty moot point, your talking about the most minute optimizations. 
Is this a performance bottleneck? no

Reply:
It's not a moot point. Why run the deep compare for all props/objects all the time?

Reply:
It is, you're talking about something so inconsequential performance-wise it IS a moot point.

Reply:
Are we discussing whether it's a moot point or that we unnecessarily run multiple recurring functions 😄

Reply:
You mentioned performance, that was the discussion. 
This function is essentially a lodash wrapper with some options, of which is about as efficient as we can get. 

Also, the running of said functions comparing some strings or simple objects to one another is microscopic in difference performance wise.

We are talking about running this function once when we edit a single owner.
If we were talking about running 100's of thousands owners through this function in the same instant, we could probably look into optimizing sure, but this function here would be the least of our concerns in that scenario. 

All of this aside, the function returns on the first detected change it does not iterate through them all, so i'm not sure where you are coming from here honestly.

Reply:
Thanks for taking the time to clarify the context. Sounds good to me.